be/src/exec/pipeline/sink/schema_table_sink_operator.cpp

```diff
@@ -0,0 +1,130 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "agent/master_info.h"
+#include "common/configbase.h"
+#include "exec/pipeline/pipeline_driver_executor.h"
+#include "exec/pipeline/sink/schema_table_sink_opeartor.h"
+#include "exprs/expr_context.h"
+#include "http/action/update_config_action.h"
+
+namespace starrocks::pipeline {
+Status SchemaTableSinkOperator::prepare(RuntimeState* state) {
+    RETURN_IF_ERROR(Operator::prepare(state));
+    return Status::OK();
+}
+
+void SchemaTableSinkOperator::close(RuntimeState* state) {
+    Operator::close(state);
+}
+
+bool SchemaTableSinkOperator::need_input() const {
+    return !_is_finished;
+}
+
+bool SchemaTableSinkOperator::is_finished() const {
+    return _is_finished;
+}
+
+Status SchemaTableSinkOperator::set_finishing(RuntimeState* state) {
+    _is_finished = true;
+    return Status::OK();
+}
+
+bool SchemaTableSinkOperator::pending_finish() const {
+    return false;
+}
+
+Status SchemaTableSinkOperator::set_cancelled(RuntimeState* state) {
+    _is_finished = true;
+    return Status::OK();
+}
+
+StatusOr<ChunkPtr> SchemaTableSinkOperator::pull_chunk(RuntimeState* state) {
+    return Status::InternalError("Shouldn't pull chunk from SchemaTableSinkOperator");
+}
+
+Status SchemaTableSinkOperator::push_chunk(RuntimeState* state, const ChunkPtr& chunk) {
+    // Same as ResultSinkOperator, The memory of the output result set should not be counted in the query memory,
+    // otherwise it will cause memory statistics errors.
```

Comment:
Why this memory should be excluded? And I don't see any code for this exclusion.